lib/experimental/stylebook/classic-screen.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Set up a screen to show stylebook for classic themes.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Add a Styles submenu under the Appearance menu
+ * for Classic themes.
+ *
+ * @global array $submenu
+ */
+function gutenberg_add_styles_submenu_item() {
```

Comment:
Since this PR is not developing an experimental feature, how about moving this function to the `lib/compat/wordpress-6.8/site-editor.php`?